package/Dockerfile.submariner-libreswan-git

```diff
@@ -0,0 +1,41 @@
+FROM registry.fedoraproject.org/fedora:32
+
+RUN dnf -y install --nodocs git audit-libs-devel bison curl-devel fipscheck-devel flex \
+    gcc ldns-devel libcap-ng-devel libevent-devel \
+    libseccomp-devel libselinux-devel make nspr-devel nss-devel \
+    pam-devel pkgconfig systemd-devel unbound-devel xmlto rpm-build \
+    openldap-devel hostname nss-tools
+
+
+WORKDIR /root
+
+RUN git clone https://github.com/libreswan/libreswan
+RUN cd libreswan && git checkout main
+RUN cd libreswan && \
+    sed -i -e "s/^%global with_cavstests 1*/%global with_cavstests 0/g" packaging/fedora/libreswan.spec 
+
+RUN cd libreswan && make git-rpm 
+
+FROM registry.fedoraproject.org/fedora-minimal:32
+
+WORKDIR /var/submariner
+
+# iproute and iptables are used internally
+# libreswan or strongswan provide IKE
+# procps-ng is needed for sysctl
+RUN microdnf -y install --nodocs --setopt=install_weak_deps=0 \
+             iproute iptables libreswan strongswan procps-ng && \
+    microdnf -y clean all
+
+COPY --from=0 /root/rpmbuild/RPMS/x86_64/* /root/
+RUN microdnf -y remove libreswan
```

Comment:
Do we need to remove `libreswan` before upgrading it?

Reply:
For some reason libreswan messes up the version packages and creates 3.30-xxx instead of 3.32-xxxxx  and using rpm to install didn't handle the dependencies, microdnf didn't suport rpm install... but hopefully this goes away with fedora:32 no-minimal.